.yamato/project.metafile

```diff
@@ -2,6 +2,8 @@
 # for validation
 test_editors:
   - 2021.1
+  - 2021.2
```

Comment:
These changes were already made in the release branch - they're pending merge back to develop so it'd make that merge easier if you don't add these here :)